assets/js/data/utils/process-error-response.ts

```diff
@@ -124,7 +124,7 @@ const processInvalidParamResponse = (
  */
 export const processErrorResponse = (
 	response: ApiErrorResponse,
-	context: string | undefined
+	context?: string | undefined
```

Comment:
This change will fix the TypeScript error present in the `assets/js/base/context/providers/cart-checkout/checkout-processor.ts` file for not passing [two arguments](https://github.com/woocommerce/woocommerce-blocks/blob/8932bc96ad8588e3feda2eba89014d1d19d92646/assets/js/base/context/providers/cart-checkout/checkout-processor.ts#L292).